lib/validators/movies/list.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).lowercase().optional(),
+  release_year: Joi.number().integer().min(1878).max(9999).optional()
```

Comment:
Since these are just for the filters, we can probably be a little bit more relaxed about the schema here. In the create validator we wanted to enforce a character limit on the `title`, for example, since it would actually be saved to the database. 

Here, however, we are just using it as part of a query. If the `title` provided is obviously too long, the query will just turn up no results and no harm done.